src/main/java/com/ericsson/ei/subscriptionhandler/InformSubscription.java

```diff
@@ -0,0 +1,114 @@
+/*
+    Copyright 2017 Ericsson AB.
+    For a full list of individual contributors, please see the commit history.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+package com.ericsson.ei.subscriptionhandler;
+
+import javax.annotation.PostConstruct;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+import com.ericsson.ei.mongodbhandler.MongoDBHandler;
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * This class represents the REST POST notification mechanism and the alternate
+ * way to save the aggregatedObject details in the database when the
+ * notification fails.
+ * 
+ * @author xjibbal
+ * 
+ */
+
+@Component
+public class InformSubscription {
```

Comment:
Change name to InformSubscriber since it does not sound logical to inform a subscription.